cgi-bin/ljlib.pl

```diff
@@ -129,6 +129,23 @@ BEGIN
 use DW::TaskQueue;
 use DW::BlobStore;
 
+# Load more modules so that we get as much advantage out of prefork
+# memory allocation as possible (as well as moving as much loading cost
+# to startup time)
+BEGIN {
+    # Do not run if we're in a test
+    unless ($LJ::_T_CONFIG) {
```

Comment:
These moved here because this is basically what mod_perl.pl was doing on startup and I needed to replicate it so that Plack would benefit from the preloading before startup/forking.